openedx/features/course_experience/templates/course_experience/course-outline-fragment.html

```diff
@@ -35,24 +35,51 @@ <h3 class="section-title">${ section['display_name'] }</h3>
+                                                                ${ _("(Prerequisite required)") }
+                                                            </span>
+                                                        </span>
+                                                    </span>
+                                                % endif 
+                                            % endif
                                         <div class="details">
-                                            ## There are behavior differences between rendering of subsections which have
-                                            ## exams (timed, graded, etc) and those that do not.
-                                            ##
-                                            ## Exam subsections expose exam status message field as well as a status icon
-                                            <%
-                                                if subsection.get('due') is None:
-                                                    # examples: Homework, Lab, etc.
-                                                    data_string = subsection.get('format')
+
+                                        ## There are behavior differences between rendering of subsections which have
+                                        ## exams (timed, graded, etc) and those that do not.
+                                        ##
+                                        ## Exam subsections expose exam status message field as well as a status icon
+                                        <%
+                                            if subsection.get('due') is None:
+                                                # examples: Homework, Lab, etc.
+                                                data_string = subsection.get('format')
+                                            else:
+                                                if 'special_exam_info' in subsection:
+                                                    data_string = _('due {date}')
                                                 else:
-                                                    if 'special_exam_info' in subsection:
-                                                        data_string = _('due {date}')
-                                                    else:
-                                                        data_string = _("{subsection_format} due {{date}}").format(subsection_format=subsection.get('format'))
-                                            %>
-                                            % if subsection.get('format') or 'special_exam_info' in subsection:
+                                                    data_string = _("{subsection_format} due {{date}}").format(subsection_format=subsection.get('format'))
+                                        %>
+                                        % if subsection.get('format') or 'special_exam_info' in subsection:
```

Comment:
lines 41-82 look like just spacing changes, not real changes. might make sense to revert those so it doesn't show as diff